examples/102.purs

```diff
@@ -0,0 +1,34 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Class (liftEff)
+import Control.Monad.Eff.Console (CONSOLE, log)
+
+import Pipes (yield)
+import Pipes.Core ((//>), runEffect, Producer_)
+
+type Result = Int
+
+myLog
+  :: forall m
+   . Monad m
+  => String
+  -> Producer_ String m Unit
+myLog = yield
+
+foo
+  :: forall eff
+   . Int  
+  -> Int
+  -> Producer_ String (Eff eff) Result
+foo x y = do
+  myLog $ "About to compute: " <> show x <> " + " <> show y
+  r <- pure $ x + y
+  myLog $ "Computed: " <> show x <> " + " <> show y <> " = " <> show r
+  pure r
```

Comment:
And also in my particular case I need `Producer Json m Unit` as I am using custom logging effect. Can do that as another example later.